operator/api/vectorized/v1alpha1/console_enterprise_types.go

```diff
@@ -179,6 +179,8 @@ type SecretStore struct {
 	GCPSecretManager *SecretManagerGCP        `json:"gcpSecretManager,omitempty"`
 	AWSSecretManager *SecretManagerAWS        `json:"awsSecretManager,omitempty"`
 	KafkaConnect     *SecretStoreKafkaConnect `json:"kafkaConnect,omitempty"`
+	// Scopes is a list of supported secret scopes
+	Scopes []string `json:"scopes,omitempty"`
```

Comment:
Should this be reconciled within `genSecretStore()` ?

Reply:
yes, thanks!